webapp/app/plugins/controller.js

```diff
@@ -108,9 +110,15 @@ treeherder.controller('PluginCtrl', [
                         _.has(buildapi_artifact[0], 'blob')){
                         // this is needed to cancel/retrigger jobs
                         $scope.artifacts.buildapi = buildapi_artifact[0];
-                        $scope.buildbotJobname = $scope.artifacts.buildapi.blob.buildername;
-                        setBuildernameHref($scope.buildbotJobname);
+                        $scope.jobLinkText = $scope.artifacts.buildapi.blob.buildername;
+                    } else {
+                        $scope.jobLinkText = $scope.job.job_type_name  + " " +
+                                             $scope.job.job_type_symbol ;
                     }
+                    $scope.searchStr = $scope.jobLinkText + " " +
+                                       $scope.job.signature;
+                    setJobSearchStrHref($scope.searchStr);
```

Comment:
Filtering on job type, symbol and signature seems a little bit redundant. What about using only the signature?